src/molecules/InputForm/InputForm.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from '../../atoms/Button/Button';
+import InputField from '../../atoms/InputField/InputField';
+
+import "./InputForm.css";
+
+const InputForm = ({
+    btnText, 
+    btnType, 
+    inputType, 
+    inputValue, 
+    onInputChange, 
+    onFormSubmit
+}) => {
+  
+    return (
+        <form onSubmit={onFormSubmit}>
```

Comment:
We can remove onSumbit from here as of now because we will drive using actions like onClick